CMakeLists.txt

```diff
@@ -22,9 +22,9 @@ endif()
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 set(CMAKE_Swift_MODULE_DIRECTORY ${CMAKE_BINARY_DIR}/swift)
 
-find_package(dispatch QUIET)
-find_package(Foundation QUIET)
-find_package(XCTest QUIET)
+find_package(dispatch QUIET CONFIG REQUIRED)
+find_package(Foundation QUIET CONFIG REQUIRED)
+find_package(XCTest QUIET CONFIG REQUIRED)
```

Comment:
I'm worried about this part of the change.  With this it is impossible to actually build with a prebuilt toolchain image.  The `CONFIG` is fine, the `REQUIRED` is the piece that worries me.

Reply:
I'll remove both the QUIET and the REQUIRED then. My main concern was that it was silently failing and was a pain to debug why.

Reply:
The thing is that it isn't an absolutely required thing - so the non `QUIET` form could be confusing too.  But I would be fine with it being non-quiet as long as it is not required.